src/WebWindow.Blazor/ComponentsDesktop.cs

```diff
@@ -124,6 +123,7 @@ private static async Task RunAsync<TStartup>(IPC ipc, CancellationToken appLifet
             serviceCollection.AddSingleton<NavigationManager>(DesktopNavigationManager.Instance);
             serviceCollection.AddSingleton<IJSRuntime>(DesktopJSRuntime);
             serviceCollection.AddSingleton<INavigationInterception, DesktopNavigationInterception>();
+            serviceCollection.AddSingleton(WebWindow);
```

Comment:
I'd prefer not to add this as a DI service, because applications might have multiple windows.

Reply:
Actually never mind. I realised this represents the root window, of which there is always exactly one. So I agree it does make sense to have this as a DI service.